packages/client3/pages/backer/all-artist-pools/index.page.tsx

```diff
@@ -0,0 +1,125 @@
+import { gql, useQuery } from "@apollo/client";
+import { BigNumber } from "ethers";
+
+import { PoolCard } from "@/components/dashboard/pool-card";
+import {
+  TabButton,
+  TabContent,
+  TabGroup,
+  TabList,
+  TabPanels,
+} from "@/components/design-system";
+import { SupportedCrypto } from "@/lib/graphql/generated";
+import { TRANCHED_POOL_STATUS_FIELDS } from "@/lib/pools";
+
+const TRANCHED_POOL_CARD_FIELDS = gql`
+  ${TRANCHED_POOL_STATUS_FIELDS}
+  fragment TranchedPoolCardFieldsTemp on TranchedPool {
+    id
+    name @client
+    category @client
+    icon @client
+    artist @client
+    id
+    creditLine {
+      maxLimit
+    }
+    juniorTranches {
+      id
+      lockedUntil
+      principalDeposited
+    }
+  }
+`;
```

Comment:
What I did in another PR was to make a queries folder and export my queries from there. I felt like things get super cluttered with them all in one file.